modules/ROOT/pages/data-bridge.adoc

```diff
@@ -1,4 +1,4 @@
-= Set Up the Data Bridge
+= Manual Setup: Set Up the Data Bridge
```

Comment:
We could move this to the Manual setup section

Reply:
And ww should move the Prepare Identity Provider Data to the prerequisites and use a pointer here.

Reply:
Done.